charts/rhtap-dh/templates/plugins.yaml

```diff
@@ -33,6 +33,23 @@ data:
         package: ./dynamic-plugins/dist/backstage-plugin-techdocs
       - disabled: false
         package: ./dynamic-plugins/dist/backstage-plugin-kubernetes
+      - disabled: false
```

Comment:
The new plugins need to be gated based on the presence of the `rhtap-jenkins-integration`  secret. See how it's done for argocd.